src/components/Constants/LoadConstantsMock.ts

```diff
@@ -0,0 +1,860 @@
+                },
+                {
+                  result: 'COMPLETED',
+                  task: 'PARTNER_CARE_EMAIL',
+                },
+                {
+                  result: 'COMPLETED',
+                  task: 'PARTNER_CARE_TEXT_MESSAGE',
+                },
+                {
+                  result: 'COMPLETED',
+                  task: 'PARTNER_CARE_SOCIAL_MEDIA',
+                },
+                {
+                  result: 'COMPLETED',
+                  task: 'PARTNER_CARE_IN_PERSON',
+                },
+                {
+                  result: 'COMPLETED',
+                  task: 'PARTNER_CARE_THANK',
+                },
+                {
+                  result: 'COMPLETED',
+                  task: 'PARTNER_CARE_DIGITAL_NEWSLETTER',
+                },
+                {
+                  result: 'COMPLETED',
+                  task: 'PARTNER_CARE_PHYSICAL_NEWSLETTER',
+                },
+                {
+                  result: 'COMPLETED',
+                  task: 'PARTNER_CARE_PRAYER_REQUEST',
+                },
+                {
+                  result: 'COMPLETED',
+                  task: 'PARTNER_CARE_UPDATE_INFORMATION',
+                },
+                {
+                  result: 'COMPLETED',
+                  task: 'PARTNER_CARE_TO_DO',
+                },
+              ],
+              name: 'PARTNER_CARE_COMPLETED',
+              suggestedContactStatus: null,
+              suggestedNextActions: [
+                'PARTNER_CARE_PHONE_CALL',
+                'PARTNER_CARE_EMAIL',
+                'PARTNER_CARE_TEXT_MESSAGE',
+                'PARTNER_CARE_SOCIAL_MEDIA',
+                'PARTNER_CARE_IN_PERSON',
+                'PARTNER_CARE_THANK',
+                'PARTNER_CARE_DIGITAL_NEWSLETTER',
+                'PARTNER_CARE_PHYSICAL_NEWSLETTER',
+                'PARTNER_CARE_PRAYER_REQUEST',
+                'PARTNER_CARE_UPDATE_INFORMATION',
+                'PARTNER_CARE_TO_DO',
+              ],
+            },
+          ],
+          tags: null,
+        },
+        tasks: [
+          'PARTNER_CARE_PHONE_CALL',
+          'PARTNER_CARE_EMAIL',
+          'PARTNER_CARE_TEXT_MESSAGE',
+          'PARTNER_CARE_SOCIAL_MEDIA',
+          'PARTNER_CARE_IN_PERSON',
+          'PARTNER_CARE_THANK',
+          'PARTNER_CARE_DIGITAL_NEWSLETTER',
+          'PARTNER_CARE_PHYSICAL_NEWSLETTER',
+          'PARTNER_CARE_PRAYER_REQUEST',
+          'PARTNER_CARE_UPDATE_INFORMATION',
+          'PARTNER_CARE_TO_DO',
+        ],
+      },
+      {
+        contactStatuses: [
+          'NOT_INTERESTED',
+          'UNRESPONSIVE',
+          'NEVER_ASK',
+          'RESEARCH_ABANDONED',
+          'EXPIRED_REFERRAL',
+        ],
+        id: 'ARCHIVE',
+        name: 'Archive',
+        results: {
+          resultOptions: [],
+          tags: null,
+        },
+        tasks: [],
+      },
+    ],
+  },
+} as unknown as LoadConstantsQuery;
```

Comment:
We need to remove `as unkown`. We need to make this object match the shape of `LoadConstantsQuery` if it doesn't already.